Compiling on Appveyor failed without this. Seems like back_inserter is not part of std in VS15?

diff --git a/src/obmolecformat.cpp b/src/obmolecformat.cpp
index c270405eb..75141da1f 100644
--- a/src/obmolecformat.cpp
+++ b/src/obmolecformat.cpp
@@ -23,6 +23,7 @@ GNU General Public License for more details.
 #endif

 #include <algorithm>
+#include <iterator>

 using namespace std;
 namespace OpenBabel
